packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -1712,6 +1766,19 @@ export default class Meeting extends StatelessWebexPlugin {
         }
       });
 
+    this.locusInfo.on(LOCUSINFO.EVENTS.CONTROLS_MEETING_BREAKOUT_UPDATED,
+      ({breakout}) => {
+        this.breakouts.updateBreakout(breakout);
```

Comment:
do we need to call this on `this.breakouts.on(BREAKOUTS.EVENTS.MEMBERS_UPDATE, () => {` too?

Reply:
no. This is receiving information from locus and updating the breakouts with that information. After doing that it triggers the event so the app consuming this knows something has changed. The other event is listening to changes that have already happened regarding the members of the breakouts. No need to update anything else on the breakouts, simply to inform the consuming app that something has changed.